### PR TITLE
marker unreal units distance

### PR DESCRIPTION
**Pull request by Fredaly**
add option to edit marker distance on spline with normalized distance or cm distance

### CODE REVIEW

File: Source/SplineFollow/Public/Components/SFSplineMarkers.h [hunk 1]
```diff
@@ -76,21 +76,38 @@ struct SPLINEFOLLOW_API FSFSplineMarkerInfos
...
     {}
 
+    void UpdateDistances( float length );
+
     UPROPERTY( EditAnywhere, BlueprintReadOnly )
     ESFSplineMarkerType Type;
 
-    UPROPERTY( EditAnywhere, BlueprintReadOnly, meta = ( UIMin = 0.0f, UIMax = 1.0f, ClampMin = 0.0f, ClampMax = 1.0f, EditCondition = "Type == ESFSplineMarkerType::Single" ) )
+    UPROPERTY( EditAnywhere, meta = ( InlineCategoryProperty ) )
+    bool bNormalizedDistance = true;
```
Comment (Frickerson) [inline Source/SplineFollow/Public/Components/SFSplineMarkers.h]: uint8

**Review by Frickerson — CHANGES_REQUESTED** [review]

File: Source/SplineFollow/Public/Components/SFSplineMarkers.h (hunk 1 above)
Comment (Fredaly) [reply Source/SplineFollow/Public/Components/SFSplineMarkers.h]: done

File: Source/SplineFollow/Public/Components/SFSplineMarkers.h [hunk 2]
```diff
@@ -75,6 +75,7 @@ struct SPLINEFOLLOW_API FSFSplineMarkerInfos
 
     FSFSplineMarkerInfos() :
         Type( ESFSplineMarkerType::Single ),
+        bNormalizedDistance( 1 ),
```
Comment (Frickerson) [inline Source/SplineFollow/Public/Components/SFSplineMarkers.h]: I would init it with "true" instead of "1"

**Review by Frickerson — CHANGES_REQUESTED** [review]

File: Source/SplineFollow/Public/Components/SFSplineMarkers.h (hunk 2 above)
Comment (Fredaly) [reply Source/SplineFollow/Public/Components/SFSplineMarkers.h]: done

**Review by Frickerson — APPROVED** [review]

**Review by Frickerson — APPROVED** [review]

**Pull request merged**